### DISCUSSION TITLE
Brew dependencies versions

### DISCUSSION THREAD

**Discussion by abebars** [goreleaser/goreleaser#3283] [Q&A] [answered]
In Homebrew formulas, you can specify the version of the dependency https://docs.brew.sh/Formula-Cookbook#specifying-other-formulae-as-dependencies. But looking at the config [here](https://github.com/goreleaser/goreleaser/blob/7f2b50b94d1c425bf41c83124ed4a4bdab1fd3a1/pkg/config/config.go#L82)  I don't see where we can set this value.

**Comment by caarlos0** [comment, marked as answer]
I don't think its possible right now, no, can you open a feature request? Thanks

**Reply by abebars** [reply]
Yeah, for sure, I can try to do that and may be able to have a pull request around that. If you can, give me some hints on where to start.
